packages/client/src/pages/Forum/ForumTopic/index.tsx

```diff
@@ -0,0 +1,86 @@
+import {useState} from 'react';
+import {Avatar} from 'client/src/components/Avatar';
+import {Button} from 'client/src/components/Button';
+import {useInput} from 'client/src/hooks/useInput';
+import {ForumTopicHeader} from './ForumTopicHeader/ForumTopicHeader';
+import {Message} from './Message';
+import styles from './ForumTopic.module.scss';
+
+const dataMock = [
+	{id: 1, isOwner: true, text: 'Хей! Привет, мы рады попрепетвовать тебя на нашем форуме!'},
+	{id: 2, isOwner: false, text: 'Как ты прошел этот непроходимый уровень?'}
+];
+type TDataMock = {id: number; isOwner: boolean; text: string};
+
+export const ForumTopic = () => {
+	const [isFocusing, setIsfocusing] = useState(false);
```

Comment:
Мелочь но все же думаю будет правильней:
```suggestion
	const [isFocusing, setIsFocusing] = useState(false);
```

Reply:
Спасибо, проглядел)